lib/src/source_visitor.dart

```diff
@@ -2034,7 +2022,6 @@ class SourceVisitor extends ThrowingAstVisitor {
     token(node.rightParenthesis);
     builder.unnest();
 
-    @override
```

Comment:
How is this related to the rest of the CL?

Did the analyzer classes remove this method?

Reply:
It's not related, I just noticed this in passing. This is a _local_ function. That metadata annotation never did anything useful. I must have accidentally copied it from a method at some point.